src/Footer.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+const StyledFooter = styled.footer`
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+`;
+
+const StyledLink = styled.a`
+  text-decoration: none;
+  color: #636363;
+  font-size: 14px;
+  padding: 0 130px 0 130px;
```

Comment:
Setting fixed padding here will probably cause problems with responsiveness. Alternatively you can set `max-width` in combination with `margin: auto` to the `StyledFooter` component.